src/book/03-guides/03-hardware/01-working-with-robots.mdx

```diff
@@ -5,3 +5,62 @@ title: Working with Robots
 description: How to safely handle and work with robots.
 slug: /guides/hardware/working-with-robots
 ---
+
+## Aligning an Ankle
+
+An alignment of a ankle may need to be done if the foot and the leg of the robot aren't perpendicular when standing. 
+An ankle alignment may also be necessary when the foot has come loose. 
+
+To align an ankle correctly perform the following instructions:
+
+1. On the inner part of the robots leg, unscrew the screws holding the plastic casing to the the ankle and knee servo. 
+Remove the plastic casing, as seen in the following image:
+![leg cover](./images/ankle_leg/leg_cover.jpg 'leg cover')
+
+2. There should be a silver disk, the following photo is a close up:
+![servo close up](./images/ankle_leg/servo_close.jpg 'servo close up')
+There are three small indents on the silver disk, two on one side and one on the other. The single dot should be pointing towards the ground when the robot is standing and the other two dots towards the head.
+These three dots form an arrow, this arrow should be pointing downwards when the robot is standing.
```

Comment:
This description is open ended when describing any joint. It would be better to specify the actual hole on the printed part that each arrow should point to in the zero position

Reply:
@tayloryoung6396  not sure what you mean by " specify the actual hole on the printed part " not sure what printed part on the foot/ankle you are referring to

Reply:
Maybe a render of the printed part, with a red circle around the specific hole that the horn points towards. Could be in a subsection with renders for each part